examples/storage/cassandra/cassandra-statefulset.yaml

```diff
@@ -34,6 +34,10 @@ spec:
           capabilities:
             add:
               - IPC_LOCK
+        lifecycle:
+          preStop:
+            exec:
+              command: ["/bin/sh", "-c", "PID=$(pidof java) && kill $PID && while ps -p $PID > /dev/null; do sleep 1; done"]
```

Comment:
This has been merged ages ago but isn't it better to call nodetool drain? It should stop accepting new data and flush on disk.

cc @chrislovecnm @jondubois